### PR TITLE
Map server commands

### PR DESCRIPTION
**Pull request by Goodlyay**
Re-opening on another branch this time.
Mapped classes directly related to server commands.

### CODE REVIEW

File: mappings/net/minecraft/CommandDispatcher.mapping [hunk 1]
```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_39 net/minecraft/CommandDispatcher
```
Comment (DanyGames2014) [inline mappings/net/minecraft/CommandDispatcher.mapping]: This isn't in any package, would expect it to be in some kind of `net.minecraft.server.command` package

File: mappings/net/minecraft/server/Command.mapping [hunk 2]
```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_38 net/minecraft/server/Command
+	FIELD field_159 message Ljava/lang/String;
+	FIELD field_160 commandDispatcher Lnet/minecraft/class_39;
```
Comment (DanyGames2014) [inline mappings/net/minecraft/server/Command.mapping]: [dispatcher](https://maven.fabricmc.net/docs/yarn-1.20-pre4+build.2/net/minecraft/server/command/CommandManager.html#dispatcher) in yarn

File: mappings/net/minecraft/server/Command.mapping [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+	METHOD <init> (Ljava/lang/String;Lnet/minecraft/class_39;)V
```
Comment (DanyGames2014) [inline mappings/net/minecraft/server/Command.mapping]: Would be good to also map argument 2 as dispatcher while you're here
obrazek

File: mappings/net/minecraft/server/ServerCommandHandler.mapping [hunk 4]
```diff
@@ -0,0 +1,19 @@
+CLASS net/minecraft/class_426 net/minecraft/server/ServerCommandHandler
+	FIELD field_1686 logger Ljava/util/logging/Logger;
+	FIELD field_1687 minecraftServer Lnet/minecraft/server/MinecraftServer;
```
Comment (DanyGames2014) [inline mappings/net/minecraft/server/ServerCommandHandler.mapping]: should be server (viz. discord convo)

File: mappings/net/minecraft/server/ServerCommandHandler.mapping [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+	METHOD <init> (Lnet/minecraft/server/MinecraftServer;)V
+		ARG 1 minecraftServer
```
Comment (DanyGames2014) [inline mappings/net/minecraft/server/ServerCommandHandler.mapping]: should be server (viz. discord convo)

File: mappings/net/minecraft/server/network/ServerPlayNetworkHandler.mapping [hunk 6]
```diff
@@ -1,4 +1,13 @@
 CLASS net/minecraft/class_11 net/minecraft/server/network/ServerPlayNetworkHandler
+	FIELD field_916 logger Ljava/util/logging/Logger;
+	FIELD field_917 connection Lnet/minecraft/class_117;
+	FIELD field_919 minecraftServer Lnet/minecraft/server/MinecraftServer;
```
Comment (DanyGames2014) [inline mappings/net/minecraft/server/network/ServerPlayNetworkHandler.mapping]: should be server (viz. discord convo)

File: mappings/net/minecraft/server/network/ServerPlayNetworkHandler.mapping [hunk 7]
```diff
@@ -1,4 +1,13 @@
...
+	FIELD field_920 serverPlayerEntity Lnet/minecraft/class_69;
```
Comment (DanyGames2014) [inline mappings/net/minecraft/server/network/ServerPlayNetworkHandler.mapping]: [player](https://maven.fabricmc.net/docs/yarn-1.20-pre4+build.2/net/minecraft/server/network/ServerPlayNetworkHandler.html#player) in yarn

File: mappings/net/minecraft/server/ServerCommandHandler.mapping [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+	METHOD method_1411 executeCommand (Lnet/minecraft/class_38;)V
+		ARG 1 command
+	METHOD method_1412 parseInt (Ljava/lang/String;I)I
+		ARG 1 intString
```
Comment (DanyGames2014) [inline mappings/net/minecraft/server/ServerCommandHandler.mapping]: probably only [string](https://maven.fabricmc.net/docs/yarn-1.20-pre4+build.2/net/minecraft/util/math/MathHelper.html#parseInt(java.lang.String,int)) in yarn, seems like the method has understandably been moved to MathHelper

**Review by DanyGames2014 — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/server/command/Command.mapping [hunk 9]
```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_38 net/minecraft/server/command/Command
+	FIELD field_159 message Ljava/lang/String;
```
Comment (calmilamsy) [inline mappings/net/minecraft/server/command/Command.mapping]: Not 100% on this, but my only suggestion is perhaps `commandText` instead, considering it cuts the `/` from the start, so it's not the whole message.

File: mappings/net/minecraft/server/command/CommandDispatcher.mapping [hunk 10]
```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_39 net/minecraft/server/command/CommandDispatcher
```
Comment (calmilamsy) [inline mappings/net/minecraft/server/command/CommandDispatcher.mapping]: This should be `CommandOutput`. This doesn't handle command logic, and simply logs and provides user feedback. Took a bit of poking around to figure this out.

File: mappings/net/minecraft/server/command/ServerCommandHandler.mapping [hunk 11]
```diff
@@ -0,0 +1,19 @@
+CLASS net/minecraft/class_426 net/minecraft/server/command/ServerCommandHandler
+	FIELD field_1686 logger Ljava/util/logging/Logger;
+	FIELD field_1687 server Lnet/minecraft/server/MinecraftServer;
+	METHOD <init> (Lnet/minecraft/server/MinecraftServer;)V
+		ARG 1 server
+	METHOD method_1411 executeCommand (Lnet/minecraft/class_38;)V
```
Comment (calmilamsy) [inline mappings/net/minecraft/server/command/ServerCommandHandler.mapping:6]: On an unrelated note: I regret looking at this method.

**Review by calmilamsy — CHANGES_REQUESTED** [review]
Otherwise, looks good at a glance.

**Review by DanyGames2014 — APPROVED** [review]

File: mappings/net/minecraft/server/command/ServerCommandHandler.mapping (hunk 11 above)
Comment (DanyGames2014) [reply mappings/net/minecraft/server/command/ServerCommandHandler.mapping:6]: Yeah me too, no wonder they rewrote pretty much the entire thing 😭

Comment (Goodlyay) [reply mappings/net/minecraft/server/command/ServerCommandHandler.mapping:6]: I also regretted looking at this method

File: mappings/net/minecraft/server/command/Command.mapping (hunk 9 above)
Comment (Goodlyay) [reply mappings/net/minecraft/server/command/Command.mapping]: A bit verbose but commandAndArgs maybe? I can't think of anything else that makes it more clear what it is

**Review by calmilamsy — APPROVED** [review]
I'm happy with these names.

**Pull request merged**